pkg/parser/enrich.go

```diff
@@ -44,7 +44,6 @@ func Loadplugin(path string) ([]EnricherCtx, error) {
 	if err != nil {
 		log.Warningf("load (fake) plugin load : %v", err)
 		c.initiated = false
-		return nil, err
```

Comment:
Why ? I think we should fatal if this fails ?

Reply:
no, the plugins are "attempted" to be loaded even if the parser itself isn't enabled.
It isn't very elegant but we have to deal with this until we transform the enrichment into proper modules/plugins ^^